src/pypromice/tx/get_msg.py

```diff
@@ -16,63 +19,60 @@ def parse_arguments_msg():
                         required=False, help='Path where to write output')    	
     parser.add_argument('-u', '--uid', default=None, type=str, 
                         required=False, help='Last uid, defined from files if not given')	
-    parser.add_argument('-m', '--mailbox', default="[Gmail]/All Mail", 
+    parser.add_argument('-m', '--mailbox', default="[Gmail]/All Mail",
```

Comment:
We just need to make sure that the mailbox folder is inputted correctly. 

In the case of accessing all mails from a google account, double quotations are needed, i.e. `'"[Gmail]/All Mail"'`, otherwise the mailbox folder is not recognised (I guess this is because of the spaces in the folder name?)